sql/parse/parse.go

```diff
@@ -144,7 +158,12 @@ func convert(ctx *sql.Context, stmt sqlparser.Statement, query string) (sql.Node
 	case *sqlparser.Insert:
 		return convertInsert(ctx, n)
 	case *sqlparser.DDL:
-		return convertDDL(n)
+		// unlike other statements, DDL statements have loose parsing by default
+		ddl, err := sqlparser.ParseStrictDDL(query)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Good catch!

Reply:
I added some engine tests for drop and expanded the ones for create. PTAL!